### ISSUE TITLE
REINFORCE actor update missing discount factor

### ISSUE THREAD

**Issue by GreatArcStudios** [facebookresearch/Pearl#46]
It seems that the REINFORCE implementation may be missing the discount factor in actor updates. The loss is just computed as:

```
loss = torch.sum(negative_log_probs * return_batch)
```

Which we can see from[ `_actor_learn_batch`](https://github.com/facebookresearch/Pearl/blob/main/pearl/policy_learners/sequential_decision_making/reinforce.py#L89). Is this intentional?

**Comment by yiwan-rl** [comment]
Yes, it is intentional. While in theory the actor's update should include the discount factor, in practice it is common to ignore it. If you are interested in more about this, there are several works studying this mismatch (e.g., https://arxiv.org/pdf/2010.01069.pdf, https://arxiv.org/abs/1906.07073).

**Comment by GreatArcStudios** [comment]
I see, thanks for the reference!
